src/config.ts

```diff
@@ -38,24 +41,146 @@ export interface Config extends AuthenticationConfig {
     // Version of the service. It can be an arbitrary string. Stackdriver
     // Profiler profiles each version of each service in each zone once per
     // minute.
-    // version defaults to the value in the environment variable GAE_VERSION.
+    // Defaults to the value in the environment variable GAE_VERSION.
     // If there is no value for GAE_VERSION, it defaults to an empty string.
     version?: string;
   };
 
   // Virtual machine instance to associate profiles with instead of the one
   // read from the VM metadata server.
+  // Defaults to an empty string.
   instance?: string;
 
   // Zone to associate profiles with instead of the one read from the VM
   // metadata server.
+  // Defaults to an empty string.
   zone?: string;
 
   // When true, CPU profiling will be disabled.
-  // The default value is false.
+  // Defaults to false.
   disableCpu?: boolean;
 
   // When true, heap profiling will be disabled.
-  // The default value is false.
+  // Defaults to false.
   disableHeap?: boolean;
 }
+
+// Default values for configuration for a profiler.
+const defaultConfig: Config = {
+  projectId: '{{projectId}}',
+  logLevel: 1,
+  serviceContext: {
+    service: undefined,
```

Comment:
teeny style nit: it would be more idiomatic to skip the property if the value is `undefined`.